custom/icds_reports/tasks.py

```diff
@@ -178,7 +181,8 @@ def run_move_ucr_data_into_aggregation_tables_task():
 
 @serial_task('move-ucr-data-into-aggregate-tables', timeout=36 * 60 * 60, queue='icds_aggregation_queue')
 def move_ucr_data_into_aggregation_tables(date=None, intervals=2):
-    date = date or datetime.utcnow().date()
+    start_time = datetime.now(pytz.utc)
```

Comment:
You could use `utcnow` and not need the `pytz` import

Reply:
That was giving an error I tried 'datetime.utcnow().astimezone(INDIA_TIMEZONE)' and gave error this 'astimezone() cannot be applied to a naive datetime'

Reply:
cant this be `datetime.now(INDIA_TIMEZONE)` thats just a pytz timezone. same with below. seems odd to make it in utc then convert

Reply:
At this place it is needed because utc was being used by the later in code.

Reply:
Ah, makes sense. looks like it can still be simplified at https://github.com/dimagi/commcare-hq/pull/24105/files#diff-b8c5a5e308a203bcaee8ed83e6837edfR642

Reply:
@calellowitz  Done.